### ISSUE TITLE
fix: Style tag is displayed in FluentSelect 3.1.1

### ISSUE THREAD

**Issue by tomokusaba** [microsoft/fluentui-blazor#805]
# 🐛 Bug Report

Fluent UI Blazor 3.1.1
.NET 6
Blazor WebAssembly

## 💻 Repro or Code Sample
```
<h1>Hello, world!</h1>

<FluentSelect Items="select" TOption="string"/>


@code {
    IQueryable<string> select = new List<string> { "Microsoft MVP for Developer Technologes", "Microsoft MVP for  AI", "Microsoft MVP for  Windows and Devices for IT" }.AsQueryable();
}
```
![image](https://github.com/microsoft/fluentui-blazor/assets/46523924/1cebf83d-ad73-4bb6-8208-0931c1ab5c3b)

https://github.com/tomokusaba/FluentUI3Dotnet6

## 🤔 Expected Behavior
I want you to apply Style.

## 😯 Current Behavior
Style tag is displayed

## 🌍 Your Environment

* OS & Device: Windows11
* Browser :Edge
* .NET and FAST Version : .NET 6 Fluent UI Blazor 3.1.1

**Comment by dvoituron** [comment]
Thank you for your feedback.
This problem seems to occur only with .NET6 (but not with .NET7 or higher).
Let me check how to solve that.

**Comment by dvoituron** [comment]
Fixed in the PR #805 and will be included in the next release.
(in the meantime, if you upgrade your project to .NET7, the problem will be solved)

**Comment by emoangelov** [comment]
Same story on .Net 8 - any updates?
OS & Device: Windows11
Browser :Edge
.NET 8
Microsoft.FluentUI.AspNetCore.Components, Version=4.3.1
